src/app/components/Text/Text.js

```diff
@@ -2,61 +2,25 @@ import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import './Text.sass';
 
-/**
-Text Component
----
-##### Default Props
-```
-children: 'hello world'
-small:    true
-medium:   false
-large:    false
-huge:     false
-```
-##### Example
-```
-<Text small>Small text</Text>
-<Text medium>Medium text</Text>
-<Text large>Large text</Text>
-```
-*/
-
-// const Text = ({ children, small, medium, large, huge, crazy }) => {
-//   let className = 'Text';
-//   if (small) className += ' small';
-//   if (medium) className += ' medium';
-//   if (large) className += ' large';
-//   if (huge) className += ' huge';
-//   if (crazy) className += ' crazy';
-//   return <div className={className}>{children || 'default text'}</div>;
-// };
-
 class Text extends Component {
   static propTypes = {
-    children: PropTypes.string,
-    small: PropTypes.bool,
-    medium: PropTypes.bool,
-    large: PropTypes.bool,
-    huge: PropTypes.bool
+    text: PropTypes.string,
+    size: PropTypes.oneOf(["small", "medium", "large", "huge", "crazy"]),
   };
 
   static defaultProps = {
-    children: 'sup baby',
-    small: true,
-    medium: false,
-    large: false,
-    huge: false
+    text: 'Lorem ipsum dolor sit amet',
+    size: "small",
   };
 
   render() {
-    let className = 'Text';
-    if (this.props.small) className += ' small';
-    if (this.props.medium) className += ' medium';
-    if (this.props.large) className += ' large';
-    if (this.props.huge) className += ' huge';
-    if (this.props.crazy) className += ' crazy';
+    let className = 'Text ';
+    if(this.props.size) {
+      className += this.props.size
+    }
```

Comment:
This one is small, but let's keep things concise with 
```
if (this.props.size) className += ` ${this.props.size}`;
```
*Note: we should append class names with a space, rather than leaving a trailing space on the default class name. So `Text` and `_small` instead of `Text_` and `small`. If there are additional modifier classes, you will lose consistency by conditionally adding spaces (i.e. `Text_`, `small`, `_bold` or `bold_`?). Prefixing every modifier class with a space gives you the consistency and doesn't use weird values for class attributes (`Text`, `_small`, `_bold`).